target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_arcadyan_vg3503j.dts

```diff
@@ -79,14 +79,14 @@
 			#address-cells = <1>;
 			#size-cells = <0>;
 
-			led-1 {
```

Comment:
led-1 satisfies "(^led-[0-9a-f]$|led)" so why change it to to non-satisfying?

Reply:
It's not gpio-led and doesn't fall into this regexp.

Reply:
Ethernet PHY [documentation](https://elixir.bootlin.com/linux/v6.15-rc5/source/Documentation/devicetree/bindings/net/ethernet-phy.yaml#L252).

Reply:
Ok, I understood.